specs/common/knative-serving-conformance-testing-workflow.md

```diff
@@ -0,0 +1,34 @@
+# Knative Conformance Workflow
+
+This document covers the workflow steps to gain knative conformance
+
+## Conformance Steps
+1. Run Manual Tests
+2. Open Github PR in knative/specs repo 
+  - Use PR template below
+  - place in the following directory structure
+    - conformance/tests/results/$spec_version/${date}-${product} directory with a metadata file and test results
+    - ex; .../v1.0/2021-12-18-ibm-code-engine/...
+  - Attach Logs from tests
+3. KTC reviews 
+  - Template
+  - logs
+4. KTC provide approval / exception / or non-approval
+5. Post to conformance matrix - consumable by the general public as a table of version, product, vendor and outcomes.
```

Comment:
What steps are needed to  'post to conformance matrix' ?

Reply:
That hasn't been setup yet.  I thought we could change this once there were decisions on how to "present' that information.

Reply:
Maybe then just add a "(to be done)" to make it clear that this is an on-going, still to be decided, process ?

Reply:
added the (tbd) to that, which sounds good.